pkg/bosh/bpmconverter/volume_factory.go

```diff
@@ -307,6 +389,10 @@ func generatePersistentVolumeClaim(manifestName string, instanceGroup *bdm.Insta
 	return persistentVolumeClaim
 }
 
+func generateEphemeralPersistentVolumeClaimName(manifestName string, instanceGroupName string) string {
```

Comment:
This is a long func name. I feel we use 'generate' a bit too much. Maybe `ephemeralName` is precise enough?

Reply:
going with `ephemeralPVCName`